Content/NPCs/Starlight/Crow.cs

```diff
@@ -66,6 +66,10 @@ public override void AI()
 		{
 			Timer++;
 
+			// This guarantees all clients get relevant info about this NPC eventually
+			if (Main.netMode == NetmodeID.Server && Main.GameUpdateCount % 60 == 0)
```

Comment:
my guess is that ALL of the current ai[] fields on the crow should not be synced and the crow may need a way of knowing when all the players are done so it can be killed by the server (kill on server when all players have starsight assuming thats synced?)